### DISCUSSION TITLE
x all commands

### DISCUSSION THREAD

**Discussion by NjuuOwO** [YimMenu/YimMenu#1985] [Q&A] [answered]
How are "do something to all" build? For example explodeall. I can't find any corresponding code to them

**Comment by ghost** [comment, marked as answer]
As far as I understand, calling the execution of a command for all depends on the corresponding parameter. If it is "false", then the command cannot be executed for everyone, and if it is not, then it is possible for everyone. I think that this is how it works on the example of OOM-kick and nf-kick, since oom cannot be called for everyone and nfkick can. `null_function_kick g_null_function_kick("nfkick", "NULL_FUNCTION_KICK", "NULL_FUNCTION_KICK_DESC", 0);` and `oom_kick g_oom_kick("oomkick", "OOM_KICK_CMD", "OOM_KICK_CMD_DESC", 0, false);` in [file](https://github.com/YimMenu/YimMenu/blob/master/src/backend/commands/player/toxic/explode_player.cpp) there is no false value. Correct me if I'm wrong
